spec.bs

```diff
@@ -3584,6 +3596,30 @@ step "Set [=this=]'s [=Request/request=] to |request|":
 
 </div>
 
+<div algorithm="iframe navigation capture adAuctionHeaders attribute patch">
+Modify the <{iframe}> element to add a
+<dfn element-attr for="iframe">adAuctionHeaders</dfn> <a spec=html>content attribute</a>.
+The IDL attribute {{HTMLIFrameElement/adAuctionHeaders}} <a spec=html>reflects</a>
+the <{iframe/adAuctionHeaders}> <a spec=html>content attribute</a>.
+
+<pre class=idl>
+partial interface HTMLIFrameElement {
+  [CEReactions] attribute boolean adAuctionHeaders;
+};
+</pre>
+</div>
+
+<div algorithm="iframe navigation patch">
+The following step will be added to the
+<a href="https://html.spec.whatwg.org/multipage/browsing-the-web.html#create-navigation-params-by-fetching">create navigation params by fetching steps</a>,

```

Comment:
Is there a way to link to this without the full URL? Is it not exported? I don't think Bikeshed can find broken links 
(in the future -- it's not broken now) with traditional `a href` links?

Reply:
maybe `<a spec="html">create navigation params by fetching</a> steps`